### PR TITLE
refactor(german-translation): corrected some simple aspects

### PR DESCRIPTION
**Pull request by mfranzke**
Fixed some simple typos in the german language file, added two missing translations and corrected some other simple stuff.

### CODE REVIEW

**Review by AJGuerin — APPROVED** [review]

**Review by dylanb — APPROVED** [review]

File: locales/de.json [hunk 1]
```diff
@@ -354,8 +354,8 @@
       "help": "In Tabellen, die größer als 3 mal 3 sind, müssen alle nichtleeren <td>-Elemente einen zugehörigen Tabellenkopf haben."
     },
     "td-headers-attr": {
-      "description": "Stellt sicher, dass jede Zelle in einer Tabelle, welche das header-Attribute verwenden, sich nur auf andere Zellen derselben Tabelle beziehen.",
-      "help": "Innerhalb eines <table>-Elementes dürfen sich Zellen, die das header-Attribut verwenden, nur auf andere Zellen derselben Tabelle beziehen."
+      "description": "Stellt sicher, dass jede Zelle in einer Tabelle, welche das headers-Attribute verwenden, sich nur auf andere Zellen derselben Tabelle beziehen.",
```
Comment (Cerberooo) [inline locales/de.json:357]:
```suggestion
      "description": "Stellt sicher, dass jede Zelle in einer Tabelle, welche das headers-Attribut verwendet, sich nur auf andere Zellen derselben Tabelle beziehen.",
```

**Comment by mfranzke** [comment]
@Cerberooo thanks for your review and feedback!

**Review by Cerberooo — APPROVED** [review]

**Review by dylanb — APPROVED** [review]

**Comment by dylanb** [comment]
reviewed security

**Comment by stephenmathieson** [comment]
@dylanb not squashing this caused problems with some of the build/release scripts. Please try to remember to squash PRs going into the `develop` branch.

**Pull request merged**